### ISSUE TITLE
DashedVMobject not returning correct output

### ISSUE THREAD

**Issue by curenosm** [3b1b/manim#1009]
When trying to draw a dashed line with custom length (width), it returns the continous version of the line.

```python
projection_line = DashedVMobject(
        Line(point_a, point_b).set_width(100).set_color(YELLOW)
    )
```

Also tried:
```python
projection_line = DashedVMobject(
        Line(point_a, point_b).set_width(100)
    ).set_color(YELLOW)
```

And:
```python
projection_line = DashedVMobject(
        Line(point_a, point_b)
    ).set_color(YELLOW).set_width(100)
```
![image](https://user-images.githubusercontent.com/18668535/80314346-a7f6fa00-87b6-11ea-9f59-b6c7976bffb3.png)

**Comment by stevenschn** [comment]
You might try DashedLine, since this is an object specifically created for your purpose. That being said, I do not know why the above does not work.

**Comment by curenosm** [comment]
Nope, it still doesn't work, at the moment of transform the DashedLine it shortens and get continous

**Comment by muhsinokcu** [comment]
i have an example . it works. i ll share with you.  once you specify the points , you need not set_width method. 
dashed_line=DashedVMobject(Line(np.array([0,0,0]),np.array([6,2,0]))).set_color(RED)
